src/common/njord.ts

```diff
@@ -1071,3 +1071,7 @@ export const throwUserTransactionError = async ({
 export const coresToUsd = (cores: number): number => {
   return cores / 100; // 100 Cores = 1 USD
 };
+
+export const usdToCores = (usd: number): number => {
+  return Math.floor(usd * 100); // 1 USD = 100 Cores
+};
```

Comment:
Checked with product so they are aware we floor so users technically loos few cents?